src/lxcfs.c

```diff
@@ -1032,6 +1039,7 @@ static void *lxcfs_init(struct fuse_conn_info *conn)
 
 #if HAVE_FUSE3
 	cfg->direct_io = 1;
+	cfg->nullpath_ok = 0;
```

Comment:
I've asked that already https://github.com/lxc/lxd-pkg-snap/issues/110#issuecomment-1377472107
Did you catch the case when this option has a non-zero value? (I mean by printf-ing it's value or using gdb memory inspection).